bundle/config/resources/volume.go

```diff
@@ -13,9 +13,25 @@ import (
 	"github.com/databricks/databricks-sdk-go/service/catalog"
 )
 
+type VolumeGrantPrivilege string
+
+const (
```

Comment:
Can you also add the Values function for this type? That'll enable automatic enum validation:
https://github.com/databricks/cli/blob/dc55c6232e30199b08dd7e025b66741453cabd4f/bundle/config/artifact.go#L16